src/olympia/accounts/serializers.py

```diff
@@ -14,11 +14,11 @@ class Meta:
         model = UserProfile
         fields = (
             'username', 'display_name', 'email',
-            'bio', 'deleted', 'display_collections',
+            'biography', 'deleted', 'display_collections',
```

Comment:
It's a pretty useless API at the moment - it only supports `GET`ing your own profile, so I doubt it.